### PR TITLE
Update URL parameters docs

### PR DESCRIPTION
**Pull request by andreeaceachir142**
# What 
- Update URL parameters docs
- Add docs about query parameters to the README file and include the new prop `supportsURLParameters`
- Rename the `hasURLParameters` to `supportsURLParameters`

### CODE REVIEW

File: packages/map-template/README.md [hunk 1]
```diff
@@ -110,3 +113,41 @@ Note that when using the React component, the properties should conform to JSX p
...
+`directionsTo` - Used like this `directionsTo=yourDestinationLocationId` when having a location ID, or like this `directionsTo=USER_POSITION` when having the user location.
+`externalIDs` - Used like this `externalIDs=0.0.1,0.0.2,0.0.3`. **Note!** You need to provide a list of comma separated values, without any spaces between the comma and the value. This will further be converted into an array of external IDs. Because of the way browsers work, you **cannot** use External IDs with the `,`, `&`, `#` and `+`, character in them, as they are interpreted by the browser in a particular way.
+`tileStyle` - Used like this `tileStyle=yourTileStyleName`. If no tile style is provided, the app will show the default tile style.
+`mapboxAccessToken` - Used like this `mapboxAccessToken=yourMapboxAccessToken`. If no mapboxAccessToken is provided, the app will default to the access token in the `.env` file. If both the `mapboxAccessToken` and the `gmApiKey` are present, the app will load a Mapbox map.
```
Comment (ammapspeople) [inline packages/map-template/README.md:135]: The `env` stuff will not work for the exported React and Web component. I don't know if that should be mentioned here.

Comment (andreeaceachir142) [reply packages/map-template/README.md:135]: That's right, I'll remove the `.env` part.

File: packages/map-template/README.md [hunk 2]
```diff
@@ -110,3 +113,41 @@ Note that when using the React component, the properties should conform to JSX p
 |`external-IDs`|`etxternalIDs`|`array`|Array of external IDs which filters the map and shows a list of locations. Because of the way browsers work, you can not use External IDs with the `,`, `&`, `#` and `+`, character in them, as they are interpreted by the browser in a particular way. |
 |`tile-style`|`tileStyle`|`string`|Name of Tile Style to display on the map. |
 |`start-zoom-level`|`startZoomLevel`|`number`|The initial zoom level of the map. |
+|`supports-URL-parameters`|`supportsURLParameters`|`bool`|Indicates if the Map Template supports URL parameters. |
+
+## Using Query Parameters
+
+The Map Template supports using query parameters for all the properties provided by the MapsIndoorsMap component.
```
Comment (ammapspeople) [inline packages/map-template/README.md:120]: Is this about the app? We should probably not mention that in readme for NPM since the app is not exported.

Comment (andreeaceachir142) [reply packages/map-template/README.md:120]: Should the whole section be moved to the general README file for the Map template?

Comment (ammapspeople) [reply packages/map-template/README.md:120]: Maybe it should be in both places.

Comment (andreeaceachir142) [reply packages/map-template/README.md]: Updated the docs to fit correctly on each page.

Comment (andreeaceachir142) [reply packages/map-template/README.md]: Also added npm links on the front page for the Map Template project and for the Midt as well

**Review by ammapspeople — APPROVED** [review]
✅

**Pull request merged**